sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/aad/AADTokenClaim.java

```diff
@@ -11,4 +11,5 @@
     public static final String NAME = "name";
     public static final String TID = "tid";
     public static final String ROLES = "roles";
+    public static final String SCP = "scp";
```

Comment:
Is this the only possible scope claim name from AAD token? Could it be scope here?

Reply:
yes,this is the only possible scp claim name from AAD token,I got it from the Microsoft docs.